udata/migrations/2017-02-25-missing-user-metrics.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Set default metric values on users
```

Comment:
This PR doesn't handle the existing values, only default ones.
Existing values are already set and working (excluding those not being updated after transfer, see #810)